### PR TITLE
Python editor text zoom

### PR DESCRIPTION
**Pull request by Amoursol**
### Purpose

Adding in the ability to increase or decrease the text font size in the Python editor, by using `CTRL` and their middle-mouse scroll. 

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

@DynamoDS/dynamo 

### FYIs

@QilongTang

### CODE REVIEW

**Comment by QilongTang** [comment]
@Amoursol The clean-up commit is not clean, please fix the build error.. FYI: you cant rename the event hanlder without updating the reference in UI code

**Review by QilongTang — APPROVED** [review]

File: src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs [hunk 1]
```diff
@@ -105,6 +105,47 @@ internal void Initialize(Guid workspaceGuid, Guid nodeGuid, string propName, str
...
+                this.UpdateFontSize(e.Delta > 0);
+                e.Handled = true;
+            }
+        }
+
+        // Reasonable max and min font size values
+        private const double FONT_MAX_SIZE = 60d;
+        private const double FONT_MIN_SIZE = 5d;
+
+        // Update function, increases/decreases by a specific increment
```
Comment (mjkkirschner) [inline src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs:125]: 1. This comment doesn't really make sense.
2.  public methods should use ///summary comments.
3. Why is this public?

Comment (Amoursol) [reply src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs:125]: @mjkkirschner it's code from elsewhere ported. For sure not ideal for Dynamo. @QilongTang is fixing it for me 😊

Comment (QilongTang) [reply src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs:125]: Oh missed this one, I will update this in another PR

**Pull request merged**